main.go

```diff
@@ -13,6 +13,17 @@ import (
 	rss "github.com/jteeuwen/go-pkg-rss"
 )
 
+var podcastEpisodePrefix = regexp.MustCompile(`^\d+: `)
+
+var sourceFeeds = []sourceFeed{
+	{uri: "https://robots.thoughtbot.com/summaries.xml", name: "Giant Robots blog"},
+	{uri: "http://simplecast.fm/podcasts/271/rss", name: "Giant Robots podcast"},
+	{uri: "http://simplecast.fm/podcasts/272/rss", name: "Build Phase podcast"},
+	{uri: "http://simplecast.fm/podcasts/282/rss", name: "The Bike Shed podcast"},
+	{uri: "http://simplecast.fm/podcasts/1088/rss", name: "Tentative podcast"},
+	{uri: "https://upcase.com/the-weekly-iteration.rss", name: "The Weekly Iteration videos"},
+}
```

Comment:
If the value were a string or number, then probably so. But slices can't be constants, so `var` is correct.